src/com/goide/quickfix/GoReplaceWithSelectStatementQuickFix.java

```diff
@@ -0,0 +1,63 @@
+/*
+ * Copyright 2013-2016 Sergey Ignatov, Alexander Zolotov, Florin Patan
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.goide.quickfix;
+
+import com.goide.psi.impl.GoElementFactory;
+import com.intellij.codeInspection.LocalQuickFixAndIntentionActionOnPsiElement;
+import com.intellij.openapi.command.WriteCommandAction;
+import com.intellij.openapi.editor.Editor;
+import com.intellij.openapi.project.Project;
+import com.intellij.psi.PsiElement;
+import com.intellij.psi.PsiFile;
+import org.jetbrains.annotations.Nls;
+import org.jetbrains.annotations.NotNull;
+import org.jetbrains.annotations.Nullable;
+
+public class GoReplaceWithSelectStatementQuickFix extends LocalQuickFixAndIntentionActionOnPsiElement {
+  public static final String QUICK_FIX_NAME = "Replace with 'select'";
+
+  public GoReplaceWithSelectStatementQuickFix(@Nullable PsiElement element) {
+    super(element);
+  }
+
+  @NotNull
+  @Override
+  public String getText() {
+    return QUICK_FIX_NAME;
+  }
+
+  @Nls
+  @NotNull
+  @Override
+  public String getFamilyName() {
+    return QUICK_FIX_NAME;
+  }
+
+  @Override
+  public void invoke(@NotNull final Project project,
+                     @NotNull PsiFile file,
+                     @Nullable("is null when called from inspection") Editor editor,
+                     @NotNull final PsiElement startElement,
+                     @NotNull PsiElement endElement) {
+    new WriteCommandAction.Simple(project, getName(), file) {
```

Comment:
startElement should be checked before, otherwise you can suddenly replace whole file with your `select` statement